src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -660,6 +660,16 @@ private static FormulaValue DateOrDateTime(IRContext irContext, int index, Formu
                 return arg;
             }
 
+            return CommonErrors.RuntimeTypeMismatch(irContext);
+        }
+
+        private static FormulaValue OptionSetOrString(IRContext irContext, int index, FormulaValue arg)
+        {
+            if (arg is StringValue || arg is OptionSetValue)
```

Comment:
>OptionSetValue [](http://example.com/codeflow?start=45&length=14)

Do we need to check that it's a string-valued option set? For example, we don't want ErrorKind.Div0 (which is a numeric option set)

Reply:
Adding this test, you're right